functions.php

```diff
@@ -123,6 +123,17 @@ function process_signins_resultset($signins) {
     return $signins_return;
 }
 
+function signins_n_days_ago($daysAgo) {
+    $oneDayInMS = 24 * 60 * 60 * 100
+    $beginOfDay = strtotime("midnight", time());
+    $dayBeginning = $beginOfDay - ($oneDayInMS * $daysAgo)
+    $dayEnd = $beginOfDay - ($oneDayInMS * ($daysAgo - 1))
+
+    $signins = signInQuery::create()->filterByCreatedAt(array('min'=>$dayBeginning), 'max'=>$dayEnd)->find();
```

Comment:
Consider looking into other ways to do this query - I know SQL can take dates as strings instead of timestamps.